components/nodemanager-service/nodemanager.go

```diff
@@ -252,12 +252,19 @@ func serve(ctx context.Context, config *config.Nodemanager, connFactory *securec
 			return err
 		}
 
+		// deprecated stuff
```

Comment:
removing deprecated stuff is difficult. If certain tasks have been queued up by the workflow, there needs to be a task executor for them otherwise the workflow cannot make progress. For this case, it was safe to make all the functions that do the work just error out, but we still have to pull those tasks and fail them forever. Or we decide we don't care and if people get stuck, they run `chef-automate dev cereal workflow kill project-update/ProjectUpdate SingletonV1`

Reply:
Ok, this is a problem with users that might have nodemanager project update tasks pending during an automate upgrade. So, we will never know if we can remove these unless we have a milestone release?

Reply:
that is mostly correct. We could create a way for cereal to automatically fail these. I'm sure there are solutions